cloudfront/wellcomelibrary.org/terraform/cf_behaviours.tf

```
@@ -101,7 +101,78 @@ locals {
     }
   ]
 
+  moh_behaviours = [
+    {
+      path_pattern     = "moh/*"
+      target_origin_id = local.wellcome_library_moh_origin.origin_id
+      headers          = []
+      cookies          = "all"
+      lambdas          = []
+
+      min_ttl     = null
+      default_ttl = null
+      max_ttl     = null
+    },
+    {
+      path_pattern     = "spas/*"
+      target_origin_id = local.wellcome_library_moh_origin.origin_id
+      headers          = []
+      cookies          = "all"
+      lambdas          = []
+
+      min_ttl     = null
+      default_ttl = null
+      max_ttl     = null
+    },
+    {
+      path_pattern     = "assets/*"
+      target_origin_id = local.wellcome_library_moh_origin.origin_id
+      headers          = []
+      cookies          = "all"
+      lambdas          = []
+
+      min_ttl     = null
+      default_ttl = null
+      max_ttl     = null
+    },
+    {
+      path_pattern     = "plugins/*"
+      target_origin_id = local.wellcome_library_moh_origin.origin_id
+      headers          = []
+      cookies          = "all"
+      lambdas          = []
+
+      min_ttl     = null
+      default_ttl = null
+      max_ttl     = null
+    },
+    {
+      path_pattern     = "scripts/*"
+      target_origin_id = local.wellcome_library_moh_origin.origin_id
+      headers          = []
+      cookies          = "all"
+      lambdas          = []
+
+      min_ttl     = null
+      default_ttl = null
+      max_ttl     = null
+    },
+    {
+      path_pattern     = "timelines/*"
+      target_origin_id = local.wellcome_library_moh_origin.origin_id
+      headers          = []
+      cookies          = "all"
+      lambdas          = []
+
+      min_ttl     = null
+      default_ttl = null
+      max_ttl     = null
+    }
+  ]
```

Given these are all the same, you could use a [for expression](https://www.terraform.io/docs/language/expressions/for.html):

```suggestion
  moh_paths = ["moh/*", "spas/*", "assets/*", "plugins/*", "scripts/*", "timelines/*"]

  moh_behaviours = [
    for path in local.moh_paths:
    {
      path_pattern     = path
      target_origin_id = local.wellcome_library_moh_origin.origin_id
      headers          = []
      cookies          = "all"
      lambdas          = []

      min_ttl     = null
      default_ttl = null
      max_ttl     = null
    }
  ]
```